editor/block-mover/test/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { BlockMover } from '../';
+
+describe( 'BlockMover', () => {
+	describe( 'basic rendering', () => {
+		it( 'should render two IconButton components with the following props', () => {
+			const blockMover = shallow( <BlockMover /> );
+			expect( blockMover.hasClass( 'editor-block-mover' ) );
+
+			const moveUp = blockMover.childAt( 0 );
+			const moveDown = blockMover.childAt( 1 );
+			expect( moveUp.type().name ).to.equal( 'IconButton' );
+			expect( moveDown.type().name ).to.equal( 'IconButton' );
+			expect( moveUp.props() ).to.include( {
+				className: 'editor-block-mover__control',
+				onClick: undefined,
+				icon: 'arrow-up-alt2',
+				'aria-disabled': undefined,
+			} );
+			expect( moveDown.props() ).to.include( {
+				className: 'editor-block-mover__control',
+				onClick: undefined,
+				icon: 'arrow-down-alt2',
+				'aria-disabled': undefined,
+			} );
+		} );
+
+		it( 'should render the up arrow with a onMoveUp callback', () => {
+			const onMoveUp = ( event ) => event;
```

Comment:
Should we do any `click` simulations with a Sinon spy?

https://github.com/airbnb/enzyme/blob/master/docs/api/ShallowWrapper/simulate.md

Reply:
With the Mocha to Jest migration it would require using [jest.fn()](http://facebook.github.io/jest/docs/en/jest-object.html#jestfnimplementation) instead to achieve similar result.